src/util/notifications_util.js

```diff
@@ -156,15 +157,20 @@ class Notificator {
         return true;
     }
 
-    async parse_connect_file(connect_filename) {
-        const filepath = path.join(this.connect_files_dir, connect_filename);
+    async parse_connect_file(connect_filename, decrypt = false) {
         let connect;
         if (this.nc_config_fs) {
-            connect = await this.nc_config_fs.get_config_data(filepath);
+            connect = await this.nc_config_fs.get_connection_by_name(connect_filename);
         } else {
+            const filepath = path.join(this.connect_files_dir, connect_filename);
             const connect_str = fs.readFileSync(filepath, 'utf-8');
             connect = JSON.parse(connect_str);
         }
+
+        if (connect.encrypt && connect.request_options_object.auth && decrypt) {
```

Comment:
Where can I see the schema of connect?
I wanted to see the properties `encrypt` and `request_options_object` and to see their type and additional explanation?

Reply:
Schema is not plausible as request_options_object (and request_options_object) are option object that follow node js API (for http agent and http request).

'encrypt' is the key id used for encryption.

Reply:
I think you can add here a comment so it will clear (at least - "'encrypt' is the key id used for encryption.").
because `decrypt` is a boolean, and I didn't know what to expect from `connect.encrypt`).

Reply:
BTW, I don't know if you want (or can change it now) but in the account schema this property is called `master_key_id` if I understand you correctly:
https://github.com/noobaa/noobaa-core/blob/e24c62765857e52715fe6a86fcca4302436db020/src/server/system_services/schemas/nsfs_account_schema.js#L42-L44